src/layers/Tracker.js

```diff
@@ -34,14 +35,18 @@ export default class Tracker {
     );
     this.canvasContext = this.canvas.getContext('2d');
 
-    this.renderCompleteRef = this.map.once('rendercomplete', () => {
-      [this.canvas.width, this.canvas.height] = this.map.getSize();
-      this.map.getTarget().appendChild(this.canvas);
-    });
-
-    this.changeSizeRef = this.map.on('change:size', () => {
-      [this.canvas.width, this.canvas.height] = this.map.getSize();
-    });
+    // Array of ol events key. We don't use a class property to be sure
+    // it's not overrided by a descendant of this class.
+    this.olEventsKeys = [
```

Comment:
Maybe stupid question, but why calling it 'olEventsKeys', if it is an array of Event.
Isn't 'olEvents' sufficient?

Reply:
because they are keys not events

Reply:
From the doc:
```
Returns:
Unique key for the listener. If called with an array of event types as the first argument, the return will be an array of keys.
```

Reply:
`olListenersKeys`  is maybe more appropriate.

Reply:
Right, then it is the good term. Sorry :)